pkg/config/config.go

```diff
@@ -64,6 +64,10 @@ func GetSystemSettingsAllowList() []string {
 		"overcommit-config",
 		"vip-pools",
 		"auto-disk-provision-paths",
+		"csi-driver-config",
+		"containerd-registry",
+		"storage-network",
```

Comment:
@tjjh89017 Could you check if it's a good idea to allow the storage-network setting when bootstrapping a cluster? Thanks!

Reply:
Please don't include storage-network.
That setting is day2 setting.

Reply:
Yes I understand and I will remove it. Shouldn't that also be the case for cluster_networks? Because I can still put that in the create-config and it's created after a deployment. However without any nics configured because the install.networks part is removed. Or is this still there for upgrade purposes?